### PR TITLE
Add connectors bundle

### LINKED ISSUE #34: Add Connectors OOTB image to docker compose
To enable users to test Camunda provided out of the box connectors we want to add them to the docker compose. As connectors are part of the core execution story we will add them to both docker compose files. The README should describe how to add secrets and custom connectors.

### PR DESCRIPTION
**Pull request by menski**
- adds connectors bundle to the core and full distribution
- connectors bundle contains all out-of-the-box connectors for Camunda 8
- add connector-secrets.txt file to inject secrets into the environment
- update README with description of the connectors

closes #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,21 @@ In addition to the local environment setup with docker-compose, you can download
...
+
+To inject secrets into the connector runtime they can be added to the
+[`.secrets`](.secrets) file inside the repository in the format `NAME=VALUE`
+per line. The secrets will then be available in the connector runtime with the
+format `secrets.NAME`.
+
+To add own connectors either create a new docker image bundling them as
+described [here](https://github.com/camunda/connector-sdk/tree/main/runtime-job-worker#docker-job-worker-runtime-image).
+
+Alternative you can mount new connector jars as volumes in to the `/opt/app` folder by adding this to the docker compose file.
```
Comment (tmetzke) [inline README.md:79]:
```suggestion
Alternatively, you can mount new connector JARs as volumes into the `/opt/app` folder by adding this to the docker-compose file.
```

File: docker-compose-core.yaml [hunk 2]
```diff
@@ -63,6 +63,18 @@ services:
       - zeebe
       - elasticsearch
 
+  connectors: # https://docs.camunda.io/docs/next/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/
```
Comment (tmetzke) [inline docker-compose-core.yaml:66]:
```suggestion
  connectors: # https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/
```

File: docker-compose.yaml [hunk 3]
```diff
@@ -87,6 +87,18 @@ services:
       - identity
       - elasticsearch
 
+  connectors: # https://docs.camunda.io/docs/next/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/
```
Comment (tmetzke) [inline docker-compose.yaml:90]:
```suggestion
  connectors: # https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/
```

**Review by tmetzke — COMMENTED** [review]
This looks pretty good 👍 
Should we add the Connectors to the list of components in the README lines 21-28 as well?
Are we waiting for the final OOTB Docker image (e.g. `0.2.0`) to merge this here?

**Comment by menski** [comment]
> Should we add the Connectors to the list of components in the README lines 21-28 as well?

Good idea I will add it to the README

> Are we waiting for the final OOTB Docker image (e.g. 0.2.0) to merge this here?

I think so yes

File: README.md [hunk 4]
```diff
@@ -63,6 +64,21 @@ In addition to the local environment setup with docker-compose, you can download
 
 Feedback and updates are welcome!
 
+
+## Connectors
+
+Both docker compose files contain our [out of the box connectors](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/).
```
Comment (tmetzke) [inline README.md:70]: Let's be consistent with the documentation.

```suggestion
Both docker-compose files contain our [out-of-the-box Connectors](https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/).
```

File: README.md [hunk 5]
```diff
@@ -63,6 +64,21 @@ In addition to the local environment setup with docker-compose, you can download
...
+
+To inject secrets into the connector runtime they can be added to the
+[`connector-secrets.txt`](connector-secrets.txt) file inside the repository in the format `NAME=VALUE`
+per line. The secrets will then be available in the connector runtime with the
+format `secrets.NAME`.
+
+To add own connectors either create a new docker image bundling them as
+described [here](https://github.com/camunda/connector-sdk/tree/main/runtime-job-worker#docker-job-worker-runtime-image).
+
+Alternatively, you can mount new connector JARs as volumes into the `/opt/app` folder by adding this to the docker-compose file.
```
Comment (tmetzke) [inline README.md:80]: Documentation agreement: `Connector(s)` should be capitalized.

```suggestion
To inject secrets into the Connector runtime they can be added to the
[`connector-secrets.txt`](connector-secrets.txt) file inside the repository in the format `NAME=VALUE`
per line. The secrets will then be available in the Connector runtime with the
format `secrets.NAME`.

To add custom Connectors either create a new docker image bundling them as
described [here](https://github.com/camunda/connector-sdk/tree/main/runtime-job-worker#docker-job-worker-runtime-image).

Alternatively, you can mount new Connector JARs as volumes into the `/opt/app` folder by adding this to the docker-compose file. Keep in mind that the Connector JARs need to bring along all necessary dependencies inside the JAR.
```

**Comment by menski** [comment]
@tmetzke for the fixes of the README, after 0.2.2 is released I will do a basic test of this setup, and then merge it. Any other hints/concerns from your side?

**Comment by tmetzke** [comment]
No other concerns from my end 👍

**Pull request merged**